tests/unit/output.js

```diff
@@ -28,6 +28,11 @@ describe('Unittests for output.js', function() {
     result = Output.isDocumentationOutput(outputData, "keyword");
     assert.isFalse(result);
 
+    // documentation is not text in a dict
```

Comment:
`in a dict` should be `in an array` right?

Reply:
it is correct. but still confusing. so updated all comments.